library/cc/engine_builder.cc

```diff
@@ -185,6 +245,15 @@ std::string EngineBuilder::generateConfigStr() {
   for (const auto& [key, value] : replacements) {
     config_builder << "- &" << key << " " << value << std::endl;
   }
+  std::vector<std::string> stat_sinks = stat_sinks_;
+  if (!stats_domain_.empty()) {
+    stat_sinks.push_back("*base_metrics_service");
```

Comment:
are we missing something along these lines in here
https://github.com/envoyproxy/envoy-mobile/blob/b5dc941df120989327e34f69509dd32b8f0b4926/library/java/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java#L305